index.html

```diff
@@ -29,23 +29,23 @@
 					setTimeout(runWhenLoadComplete, 100);
 				}
 				else {
-					populateZoomControl("school-districts-control", "texas-school-districts", "NAME", "Texas School Districts");
+					populateZoomControl("school-districts-control", "districts_1882", "NAME", "EstADAIn", "Texas School Districts");
```

Comment:
Everything relating to EstADAIn is to get the "Increase per ADA" value into the dropdown menu's metadata so that it can be used in popups.